docs/src/language_servers.md

```diff
@@ -717,6 +717,11 @@ Follow installation instructions on [LSP-tailwindcss](https://github.com/sublime
 
 Follow installation instructions on [LSP-terraform](https://github.com/sublimelsp/LSP-terraform).
 
+## Toit
+
+Install the [Toit](https://packagecontrol.io/packages/Toit) package from Package Control for syntax highlighting.
+Follow installation instructions on [LSP-toit](https://github.com/toitlang/toit/tree/master/tools/lsp/server).
```

Comment:
There are no installation instructions at https://github.com/toitlang/toit/tree/master/tools/lsp/server , you maybe put the wrong link.